usagov/usagov-2021
# usagov-1133-email-button-contrast: usagov-1133-add-class-to-button

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1133

## Description

Update to look like secondary buttons on the site. Example here: https://www.usa.gov/welfare-benefits
Fill color #CCECF2
Outline 2px solid #377f9d
## Type of Changes

- [x ] Frontend (Twig, Sass, JS)
  - Add screenshot showing what it should look like

## Testing Instructions


Goto the CEO page http://localhost/elected-officials and fill out the fields. In the results find one that has "Contact via email" button and inspect. The fill color should be #CCECF2, the outline should be 2px solid #377f9d. The hover state background color should be #377f9d.

### Requires New Config
- [x ] No

### Requires New Content
- [x ] No

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
